Dockerfile

```diff
@@ -20,7 +20,8 @@ RUN mv nri-kube-events-${TARGETOS}-${TARGETARCH} nri-kube-events
 
 USER nri-kube-events
 
-# Enable custom attributes decoration in the infra SDK
+# Enable decorating events from NRI_ env vars, such as `foo: bar` from NRI_FOO=bar.
+# https://github.com/newrelic/infra-integrations-sdk/blob/master/docs/toolset/args.md#bundled-arguments
```

Comment:
I would add a link as well to https://docs.newrelic.com/docs/integrations/kubernetes-integration/kubernetes-events/install-kubernetes-events-integration/#custom-attributes

to understand how it is used